tests/scripts/ci_perf.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3
```

Comment:
Please mark this file as executable

Reply:
Is there a way to do that from a windows machine? As you noticed the script isn't really xplat in the first place so the quick alternative is that I remove that line.

Reply:
I think there is a git.exe command to do so.